browser/src/control/Control.Tabs.js

```diff
@@ -324,9 +324,15 @@ L.Control.Tabs = L.Control.extend({
 	},
 
 	_setPart: function (e) {
-		var part =  e.currentTarget.id.match(/\d+/g)[0];
-		if (part !== null) {
-			this._setPartIndex(parseInt(part));
+		var part = e.currentTarget.id.match(/\d+/g)[0];
```

Comment:
Nit: would it be possible to avoid regex matching here? For example put the part number in a data-* attribute and then just get it instead of parsing it out of the id?